wp/wp.tex

```diff
@@ -0,0 +1,56 @@
+\documentclass[12pt]{article}
+\usepackage{pgf}
+\usepackage{setspace}
+  \setstretch{1.1}
+\usepackage[top=1.3in, left=1.4in, includefoot]{geometry}
+\usepackage{charter}
+\usepackage{listings}
+\usepackage{authblk}
+
+\definecolor{codegreen}{rgb}{0,0.6,0}
+\definecolor{codegray}{rgb}{0.5,0.5,0.5}
+\definecolor{codepurple}{rgb}{0.58,0,0.82}
+\definecolor{backcolour}{rgb}{0.95,0.95,0.92}
+
+\lstdefinestyle{mystyle}{
+	backgroundcolor=\color{backcolour},
+	commentstyle=\color{codegreen},
+	keywordstyle=\color{magenta},
+	numberstyle=\tiny\color{codegray},
+	stringstyle=\color{codepurple},
+	basicstyle=\sffamily\footnotesize,
+	breakatwhitespace=false,
+	breaklines=true,
+	captionpos=b,
+	keepspaces=true,
+	numbers=left,
+	numbersep=5pt,
+	showspaces=false,
+	showstringspaces=false,
+	showtabs=false,
+	tabsize=2
+}
+
+\lstset{style=mystyle}
+
+\title{
+  \includegraphics[height=250pt]{wp/logo.png}\\
+  \vspace{10pt}
+  \textsc{DiKTat:}\\
+  Static Code Analysis}
+\author{Yegor Bugayenko, Andrey Kuleshov, Peter Trifanov, Kumar Denis, Tsay Alexander}
+\affil{Huawei Technologies Co., Ltd. \\ System Programming Lab \\ Russian Research Institute (RRI) \\ Moscow, Russia}
+
+\begin{document}
+
+\maketitle
+
+\pagebreak
+
+\begin{abstract}
+
+DiKTat is a collection of Kotlin code style rules implemented as AST visitors on top of KTlint. Diktat warns and fixes code style errors based on configuration file. DiKTat can be extended further by adding custom rules. In this paper, we explain how DiKTat works, describe advantages and disadvantages and how it differs from other static analyzers.
```

Comment:
"diktat - is a formal strict code style (link) and a linter with a set of rules that implement this code style"
"of Ktlint framework" (paste  a link)
"code style errors and code smells"a
"highly configurable"
"can be run as command line application or with maven or gradle plugins"